File: URI-Java/1008.java
import java.io.IOException;
import java.util.Scanner;
import java.util.Locale;
 
public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Locale.setDefault(Locale.US);
        Scanner entrada = new Scanner(System.in);

        int numero = entrada.nextInt();
        int horas = entrada.nextInt();
        double valor = entrada.nextDouble();

        double salario = horas * valor;

        System.out.printf("NUMBER = %d%n", numero);
        System.out.printf("SALARY = U$ %.2f%n", salario);
        
        entrada.close();
 
    }
}

File: Go/1008.go
package main
 
import (
    "fmt"
)
 
func main() {

   	var funcionario, horas int
	var valor float64

	fmt.Scanf("%d", &funcionario)
	fmt.Scanf("%d", &horas)
	fmt.Scanf("%f", &valor)

	salario := valor * float64(horas)

	fmt.Printf("NUMBER = %d\n", funcionario)
	fmt.Printf("SALARY = U$ %.2f\n", salario)

}
